### ISSUE TITLE
Lost with this process. Please help.

### ISSUE THREAD

**Issue by zlt77** [thetatoken/guardian-mainnet-guide#82]
Hello, I'm very new to this so much of the terminology is new to me. I don't understand how this works. I signed up,  synced with the Theta network after hours. Now I'm supposed to deposit my "stake" with a QR code. What does this mean? I follow the instructions in my wallet to deposit the "stake" and I paste the QR code, but I have no amount to "stake". How do I get something to "stake"? Am I supposed to mine something or buy something? None of this is clear to a newb, and there is zero support from their website. What exactly am I supposed to do now that I'm synced to the network? How do I get Theta token?

**Comment by DavidGarry7654** [comment]
Yeah. You’re supposed to get theta tokens. You need to stake theta tokens which you’re to purchase. When you buy your theta tokens from exchange websites like binance or kucoin, you then transfer to your theta wallet then stake

**Comment by zlt77** [comment]
I see, thought this was to mine the Theta coins. The confusing part for me was because they are not on Binance at present moment. I’ve never heard of kucoin, but I’ll check that out. So what exactly is the purpose of this application then? Why would I want to deposit a stake?

**Comment by DavidGarry7654** [comment]
They’re in binance. For you not to see it on binance, I guess you’re in the US. You can use a VPN when accessing your binance account to be able to buy theta. As theta isn’t available on binance US
